### PR TITLE
[RW-5556][risk=no] UI: Make header sticky in hierarchy view

### PR DESCRIPTION
**Pull request by NehaBroad**
Screen Shot 2020-09-24 at 10 15 16 AM

Screen Shot 2020-09-28 at 11 06 20 AM

Fyi @dolbeew and @mhigbyflowers There is a follow up ticket with Lou's comment about new cohort pages

https://precisionmedicineinitiative.atlassian.net/browse/RW-5618
---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/cohort-search/tree/tree.component.tsx [hunk 1]
```diff
@@ -188,6 +196,7 @@ export const CriteriaTree = withCurrentWorkspace()(class extends React.Component
           <ClrIcon style={{color: colors.white}} className='is-solid' shape='exclamation-triangle' />
           Sorry, the request cannot be completed. Please try again or contact Support in the left hand navigation.
         </div>}
+        <div style={styles.node}>
```
[the remark this answers is not in the document]
Comment (NehaBroad) [reply ui/src/app/cohort-search/tree/tree.component.tsx]: Fixed. Also for surveys/physical measurement i have added padding of 2.5rem between search box and table. Let me know if thats fine or we need to increase/decrease it (added screen shot in PR description)

File: ui/src/app/cohort-search/tree/tree.component.tsx [hunk 2]
```diff
@@ -188,6 +195,7 @@ export const CriteriaTree = withCurrentWorkspace()(class extends React.Component
           <ClrIcon style={{color: colors.white}} className='is-solid' shape='exclamation-triangle' />
           Sorry, the request cannot be completed. Please try again or contact Support in the left hand navigation.
         </div>}
+        <div style={!this.showHeader ? styles.node : {...styles.node, borderTop: 'none'}}>
```
Comment (dolbeew) [inline ui/src/app/cohort-search/tree/tree.component.tsx:198]: I think we need to have no border here when `showHeader` is false (Physical Measurements, Surveys, Visits) like we currently do in test.

**Review by dolbeew — APPROVED** [review]
Looks good, just one minor change.

File: ui/src/app/cohort-search/tree/tree.component.tsx (hunk 2 above)
Comment (NehaBroad) [reply ui/src/app/cohort-search/tree/tree.component.tsx]: Done and have updated the screen shot in description

**Pull request merged**